### PR TITLE
Express post json

### PR DESCRIPTION
**Pull request by OSAdams**
express-post-json

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  req.body.id = nextId;
+  grades[nextId] = req.body;
+  nextId++;
+  res.sendStatus(201).json(grades[nextId]);
+});
```
Comment (TimelikeClosure) [inline express-post-json/index.js]: Make sure your middleware sends exactly one response per request.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by OSAdams** [comment]
image

File: express-post-json/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  req.body.id = nextId;
+  grades[nextId] = req.body;
+  nextId++;
+  res.sendStatus(201);
+  res.json(grades[nextId]);
+});
```
Comment (TimelikeClosure) [inline express-post-json/index.js]: Make sure you're sending exactly one response pre-request.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by OSAdams** [comment]
image

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**